### PR TITLE
2022 Week 41 release

### PR DESCRIPTION
**Pull request by mr-c**
For review & deploy Friday, October 14th ish.

This PR will update automatically as additional contributions are merged into the `main` branch.

Outstanding issues:

- [x] #260 
- [x] Update https://github.com/common-workflow-language/user_guide/blob/main/.zenodo.json with the information from AUTHORS.md (and maybe move the authors there and moving the funding acknowledgemnts to the README)
- [x] #305

### CODE REVIEW

**Comment by swzCuroverse** [comment]
@mr-c the User Guide is not listed in Zenodo.  I propose we put it in like we do for Arvados so that we can have releases listed : https://zenodo.org/record/7105209#.Y0mVTFzMKV4.  I will admit I don't know the purpose of having a zenodo.json if it isn't listed in Zenodo.

**Comment by mr-c** [comment]
> @mr-c the User Guide is not listed in Zenodo. I propose we put it in like we do for Arvados so that we can have releases listed : https://zenodo.org/record/7105209#.Y0mVTFzMKV4. I will admit I don't know the purpose of having a zenodo.json if it isn't listed in Zenodo.

???

Screenshot_20221014-190103

Points to  https://zenodo.org/record/840663 a.k.a. https://doi.org/10.5281/zenodo.840663

**Comment by swzCuroverse** [comment]
> > @mr-c the User Guide is not listed in Zenodo. I propose we put it in like we do for Arvados so that we can have releases listed : https://zenodo.org/record/7105209#.Y0mVTFzMKV4. I will admit I don't know the purpose of having a zenodo.json if it isn't listed in Zenodo.
> 
> ???
> 
> Screenshot_20221014-190103
> 
> Points to https://zenodo.org/record/840663 a.k.a. https://doi.org/10.5281/zenodo.840663

When you search Zenodo for CWL User Guide, it does not find it.   Which is a problem, then.

**Comment by swzCuroverse** [comment]
> > > @mr-c the User Guide is not listed in Zenodo. I propose we put it in like we do for Arvados so that we can have releases listed : https://zenodo.org/record/7105209#.Y0mVTFzMKV4. I will admit I don't know the purpose of having a zenodo.json if it isn't listed in Zenodo.
> > 
> > 
> > ???
> > Screenshot_20221014-190103
> > Points to https://zenodo.org/record/840663 a.k.a. https://doi.org/10.5281/zenodo.840663
> 
> When you search Zenodo for CWL User Guide, it does not find it. Which is a problem, then.

FYI - That was not appropriate way to say that the Zenodo exists.  Please try to remain civil in your responses.

**Comment by swzCuroverse** [comment]
The json file has a different DOI than the readme -- do we want the json to point to the latest release but the readme to point to the general zenodo which houses all the releases, or should they match.  Additionally, who is in charge of putting the zenodo entry to match the zenodo.json file in the GitHub repository?  I know there exist github actions for zenodo but don't know if he have them set up or not.  @kinow perhaps you know?

**Comment by mr-c** [comment]
> The json file has a different DOI than the readme

The Zenodo JSON file points to multiple related DOIs.

https://github.com/common-workflow-language/user_guide/blob/0c1df153617306d73f8acad35519487677b2bcb3/.zenodo.json#L20 

is a reference to the template we used to use from the Carpentries, and thus can be removed.

https://github.com/common-workflow-language/user_guide/blob/0c1df153617306d73f8acad35519487677b2bcb3/.zenodo.json#L29

Is a reference to the "all versions of this resource" parent DOI for the User Guide, as detailed at https://help.zenodo.org/#versioning

The badge will show the DOI for the latest release of the user.

>  -- do we want the json to point to the latest release but the readme to point to the general zenodo which houses all the releases, or should they match. Additionally, who is in charge of putting the zenodo entry to match the zenodo.json file in the GitHub repository? I know there exist github actions for zenodo but don't know if he have them set up or not. @kinow perhaps you know?

So there is no reason to update the Zenodo JSON file. Each new GitHub release automatically uploads to Zenodo.

P.S. You deleted a comment I made in response to your earlier inquiry (and that inquiry itself). Continued deletion of my comments will undermine your retaining write privileges in this and any other CWL repository.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 [![Syntax Check](https://travis-ci.org/common-workflow-language/user_guide.svg?branch=main)](https://travis-ci.org/common-workflow-language/user_guide)
 
-[User guide for CWL v1.2.0](https://www.commonwl.org/user_guide/)
+[Released User guide for CWL v1.2.0](https://www.commonwl.org/user_guide/) : This is the released version which corresponds to the release branch.  
+[Read The Docs User guide for CWL v1.2.0](https://common-workflow-languageuser-guide.readthedocs.io/en/latest/) : This is the Read The Docs(RTD) version which corresponds to the main branch.
```
Comment (mr-c) [inline README.md:6]: Use plain URL links here, please. The link text is too similar to each other.

`RTD` is defined here and never used again, so remove that abbreviation.

**Comment by swzCuroverse** [comment]
> @mr-c  I don't think this is the place to discuss this. It is inappropriate. Please refrain from using GitHub comments for this type of interaction.

**Comment by swzCuroverse** [comment]
> > The json file has a different DOI than the readme
> 
> The Zenodo JSON file points to multiple related DOIs.
> 
> https://github.com/common-workflow-language/user_guide/blob/0c1df153617306d73f8acad35519487677b2bcb3/.zenodo.json#L20
> 
> is a reference to the template we used to use from the Carpentries, and thus can be removed.
> 
> https://github.com/common-workflow-language/user_guide/blob/0c1df1536
> So there is no reason to update the Zenodo JSON file. Each new GitHub release automatically uploads to Zenodo

Do you mean the Zenodo entry? It would nice if this was documented.

**Comment by swzCuroverse** [comment]
> > The json file has a different DOI than the readme
> 
> The Zenodo JSON file points to multiple related DOIs.
> 
> https://github.com/common-workflow-language/user_guide/blob/0c1df153617306d73f8acad35519487677b2bcb3/.zenodo.json#L20
> 
> is a reference to the template we used to use from the Carpentries, and thus can be removed.
> 
> https://github.com/common-workflow-language/user_guide/blob/0c1df153617306d73f8acad35519487677b2bcb3/.zenodo.json#L29
> 
> Is a reference to the "all versions of this resource" parent DOI for the User Guide, as detailed at https://help.zenodo.org/#versioning
> 
> The badge will show the DOI for the latest release of the user.
> 
> > -- do we want the json to point to the latest release but the readme to point to the general zenodo which houses all the releases, or should they match. Additionally, who is in charge of putting the zenodo entry to match the zenodo.json file in the GitHub repository? I know there exist github actions for zenodo but don't know if he have them set up or not. @kinow perhaps you know?
> 
> So there is no reason to update the Zenodo JSON file. Each new GitHub release automatically uploads to Zenodo.
> 

I assume you mean the Zenodo entry since you have a ticket to update the  .json file.  Documentation of this and the release process would be welcome. Also, it seems like the release is a moving target. So it would be nice to know if we now have a complete list for the release.

File: README.md (hunk 1 above)
Comment (swzCuroverse) [reply README.md:6]: I believe it was added because we in tickets and other places (like chat) use RTD so people were wondering what that meant.

**Comment by mr-c** [comment]
@bamiwoaluko , @gathoni-k, and @Mackenzie-OO7: Under which name shall we list you in the [`AUTHORS.md`](https://github.com/common-workflow-language/user_guide/blob/main/AUTHORS.md) ?

**Comment by swzCuroverse** [comment]
> @bamiwoaluko , @gathoni-k, and @Mackenzie-OO7: Under which name shall we list you in the [`AUTHORS.md`](https://github.com/common-workflow-language/user_guide/blob/main/AUTHORS.md) ?

I don't think we should require people to give their formal names to be listed as authors.  I think it is fine if they do - but it shouldn't be an obligation.

**Comment by mr-c** [comment]
> > @bamiwoaluko , @gathoni-k, and @Mackenzie-OO7: Under which name shall we list you in the [`AUTHORS.md`](https://github.com/common-workflow-language/user_guide/blob/main/AUTHORS.md) ?
> 
> I don't think we should require people to give their formal names to be listed as authors. I think it is fine if they do - but it shouldn't be an obligation. People may for several reasons not be using their legal names and/or not want their names (legal or not) to be made available on a project.

Agreed, I already listed them by their GitHub handles as shown in https://github.com/common-workflow-language/user_guide/commit/c757074ae634b27d3617cce88c1e8c3bce7bdcf0 ; just giving them a chance to choose how they will be represented.

**Comment by swzCuroverse** [comment]
> > > @bamiwoaluko , @gathoni-k, and @Mackenzie-OO7: Under which name shall we list you in the [`AUTHORS.md`](https://github.com/common-workflow-language/user_guide/blob/main/AUTHORS.md) ?
> > 
> > 
> > I don't think we should require people to give their formal names to be listed as authors. I think it is fine if they do - but it shouldn't be an obligation. People may for several reasons not be using their legal names and/or not want their names (legal or not) to be made available on a project.
> 
> Agreed, I already listed them by their GitHub handles as shown in [c757074](https://github.com/common-workflow-language/user_guide/commit/c757074ae634b27d3617cce88c1e8c3bce7bdcf0) ; just giving them a chance to choose how they will be represented.

Makes sense to me.

**Comment by gathoni-k** [comment]
> @bamiwoaluko , @gathoni-k, and @Mackenzie-OO7: Under which name shall we list you in the [`AUTHORS.md`](https://github.com/common-workflow-language/user_guide/blob/main/AUTHORS.md) ?

Mary Gathoni

**Comment by bamiwoaluko** [comment]
> @bamiwoaluko , @gathoni-k, and @Mackenzie-OO7: Under which name shall we list you in the [`AUTHORS.md`](https://github.com/common-workflow-language/user_guide/blob/main/AUTHORS.md) ?

Bamiwo Aluko

**Comment by Mackenzie-OO7** [comment]
> @bamiwoaluko , @gathoni-k, and @Mackenzie-OO7: Under which name shall we list you in the [`AUTHORS.md`](https://github.com/common-workflow-language/user_guide/blob/main/AUTHORS.md) ?

Levai Mackenzie Agbara

**Comment by Mackenzie-OO7** [comment]
> > Levai Mackenzie Agbara
> 
> @Mackenzie-OO7 To confirm, should it be `Agbara` or `Ágbàrà` ?
> 
> https://github.com/common-workflow-language/user_guide/blob/864f866dbd21b77acdff6ca8627606c18677717c/AUTHORS.md?plain=1#L20

It should be `Ágbàrà`, I couldn't get those letters on my device earlier.

**Pull request merged**